spec/factories.rb

```diff
@@ -0,0 +1,16 @@
+FactoryGirl.define do
+  factory :user do
+    sequence(:name)  { |n| "Person #{n}" }
+    sequence(:email) { |n| "person_#{n}@example.com"}
+
+    factory :user_local do
+      sequence(:address)  { |n| "Person #{n}" }
+      sequence(:user_id)
+    end
+  end
+
+  factory :local do
+    sequence(:address)  { |n| "Person #{n}" }
+    sequence(:user_id)
+  end
+end
```

Comment:
It's good to create different files for each factory class. In this case, one file for `user` and other for `local`. Remember me to send some links to help you make this factory better :)